mappings/dnd5e.mapping

```diff
@@ -146,19 +146,19 @@
             let s = "";
             let a = @data.traits.weaponProf.value.map(x => game.dnd5e.config.weaponProficiencies[x]
                 || game.packs.get("dnd5e.items").index.get(game.dnd5e.config.weaponIds[x])?.name)
-                .concat(@data.traits.weaponProf.custom.split(";")).filter(x => String(x) && x.length);
-            if (a.length > 0) { s = `${s}Weapons: ${a.join(", ")}\n`; }
+                .concat(@data.traits.weaponProf.custom.split(";")).filter(x => String(x) && x && x.length);
+            if (a && a.length > 0) { s = `${s}Weapons: ${a.join(", ")}\n`; }
             a = @data.traits.armorProf.value.map(x => game.dnd5e.config.armorProficiencies[x]
                 || game.packs.get("dnd5e.items").index.get(game.dnd5e.config.armorIds[x])?.name)
-                .concat(@data.traits.armorProf.custom.split(";")).filter(x => String(x) && x.length);
-            if (a.length > 0) { s = `${s}Armor: ${a.join(", ")}\n`; }
+                .concat(@data.traits.armorProf.custom.split(";")).filter(x => String(x) && x && x.length);
+            if (a && a.length > 0) { s = `${s}Armor: ${a.join(", ")}\n`; }
             a = @data.traits.toolProf.value.map(x => game.dnd5e.config.toolProficiencies[x]
                 || game.packs.get("dnd5e.items").index.get(game.dnd5e.config.toolIds[x])?.name)
-                .concat(@data.traits.toolProf.custom.split(";")).filter(x => String(x) && x.length);
-            if (a.length > 0) { s = `${s}Tools: ${a.join(", ")}\n`; }
+                .concat(@data.traits.toolProf.custom.split(";")).filter(x => String(x) && x && x.length);
+            if (a && a.length > 0) { s = `${s}Tools: ${a.join(", ")}\n`; }
             a = @data.traits.languages.value.map(x => game.dnd5e.config.languages[x])
-                .concat(@data.traits.languages.custom.split(";")).filter(x => String(x) && x.length);
-            if (a.length > 0) { s = `${s}Languages: ${a.join(", ")}\n`; }
+                .concat(@data.traits.languages.custom.split(";")).filter(x => String(x) && x && x.length);
+            if (a && a.length > 0) { s = `${s}Languages: ${a.join(", ")}\n`; }
             return s;
```

Comment:
Could you use optional chaining for the null checks instead?

e.g.
```js
a?.length > 0
```

Otherwise, looks perfect! Thank you :smile: